setup.py

```diff
@@ -14,7 +14,7 @@
 def read(f):
     return open(os.path.join(os.path.dirname(__file__), f)).read().strip()
 
-install_requires = ['aiohttp>=3.0.1', 'typing_extensions>=3.7.4; python_version<"3.8"']
+install_requires = ['aiohttp>=3.8', 'typing_extensions>=3.7.4; python_version<"3.8"']
```

Comment:
Do we really need 3.8, or can we get away with [3.7.0](https://github.com/aio-libs/aiohttp/releases/tag/v3.7.0)?

```suggestion
install_requires = ['aiohttp>=3.7.0', 'typing_extensions>=3.7.4; python_version<"3.8"']
```

Reply:
If we could use 3.7, I wouldn't have specified 3.8. :P
3.7 has a regression that breaks core functionality in aiohttp-session, and thus also fails a bunch of tests (look around for the dozen or more mentions in various bug reports about the problem).